tests/Database/SelectFields/ComputedPropertiesTests/ComputedPropertiesTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rebing\GraphQL\Tests\Database\SelectFields\ComputedPropertiesTests;
+
+use Rebing\GraphQL\Tests\TestCaseDatabase;
+use Rebing\GraphQL\Tests\Support\Models\Post;
+use Rebing\GraphQL\Tests\Support\Models\User;
+use Rebing\GraphQL\Tests\Support\Traits\SqlAssertionTrait;
+
+class ComputedPropertiesTest extends TestCaseDatabase
+{
+    use SqlAssertionTrait;
+
+    public function testComputedProperty(): void
```

Comment:
Can you please add a phpdoc linking to the issue, similar to what I/we did in `\Rebing\GraphQL\Tests\Database\SelectFields\AlwaysRelationTests\AlwaysRelationTest::testAlwaysSingleHasManyRelationField` et al?

Reply:
sure